packit/config/package_config.py

```diff
@@ -93,7 +93,7 @@ def __init__(
         self.upstream_package_name: Optional[str] = upstream_package_name
         # this is generated by us
         self.downstream_package_name: Optional[str] = downstream_package_name
-        self.dist_git_base_url: str = dist_git_base_url or PROD_DISTGIT_URL
+        self.dist_git_base_url: str = dist_git_base_url or "https://src.stg.fedoraproject.org/"
```

Comment:
Like above and bellow, we probably cannot overwrite like this since this will change the behaviour on production.